helper/transport.go

```diff
@@ -89,7 +92,12 @@ func (t *TransportWrapper) SetTLSConfig(c *tls.Config) error {
 }
 
 func (t *TransportWrapper) RoundTrip(req *http.Request) (*http.Response, error) {
+	transportID := uuid.New().String()
+	log.Printf("Request for UUID %s sent", transportID)
 	if logging.IsDebugOrHigher() {
+		ctx := context.WithValue(req.Context(), "TransportID", transportID)
+		req = req.WithContext(ctx)
```

Comment:
I suggest we remove these lines for now since we aren't using the context value. If we do implement this in the future we will want to be sure that the key is not a string.